spec/support/mvi/find_candidate_response.xml

```diff
@@ -50,6 +50,7 @@
                 <id extension="TKIP123456^PI^200IP^USVHA^A" root="2.16.840.1.113883.4.349"/>
                 <id extension="123456^PI^200MHV^USVHA^A" root="2.16.840.1.113883.4.349"/>
                 <id extension="1234^NI^200DOD^USDOD^A" root="2.16.840.1.113883.3.364" />
+                <id extension="12345678^PI^200CORP^USVBA^A" root="2.16.840.1.113883.4.349"/>
```

Comment:
We should look at putting all of these in a consistent place. For MHV we used `spec/support/fixtures` and I made that decision mostly because I felt `spec/fixtures` is more for transactional fixtures related to active_record. I guess it doesn't really matter, but we should put them all in a consistent place.